installscripts/jazz-terraform-unix-noinstances/kinesis.tf

```diff
@@ -8,14 +8,14 @@ resource "aws_kinesis_stream" "kinesis_stream_dev" {
     "OutgoingBytes",
   ]
 
-  tags {
-    Name        = "${var.envPrefix}"
-    Application = "Jazz"
-    JazzInstance = "${var.envPrefix}"
-    Environment = "${var.tagsEnvironment}"
-    Exempt = "${var.tagsExempt}"
-    Owner = "${var.tagsOwner}"
-  }
+  tags = "${merge(var.additional_tags, map(
+          "Name", "${var.envPrefix}",
+          "Application", "Jazz",
+          "JazzInstance", "${var.envPrefix}",
+          "Environment", "${var.tagsEnvironment}",
+          "Exempt", "${var.tagsExempt}",
+          "Owner", "${var.tagsOwner}"
+          ))}"
 }
```

Comment:
Can you look at using `locals` to define the baseline set of tags *once* per file, rather than copy-pasting the same tagset multiple times in the same file?

Should be possible and would tidy this up a bit. Terraform upstream has discussed adding "global default" tags which would help us out here, but they haven't implemented yet.

Reply:
Done

Reply:
Nice!